Multisig/UI/Settings/OwnerKeyManagement/OnboardingImportOwnerKeyViewController.swift

```diff
@@ -13,9 +13,21 @@ class OnboardingImportOwnerKeyViewController: UIViewController {
     @IBOutlet weak var tableView: UITableView!
     private var nextButton: UIBarButtonItem!
 
-    private let cards = [(UIImage(named: "ico-onbaording-import-key-1"), "How does it work?", "Enter the private key or seed phrase of your owner key controlling your Safe. Your owner key will be imported into this app. You can then confirm proposed transactions on the go."),
-                         (UIImage(named: "ico-onbaording-import-key-2"), "How secure is that?", "We only store your private key. We do not store your seed phrase in the app."),
-                         (UIImage(named: "ico-onbaording-import-key-3"), "Is my wallet supported?", "You can use your seed phrase from MetaMask or hardware wallet. We support private keys from any wallet in a hexadecimal 64 characters format. Importing the key will not show the balance for Ether or other assets from your MetaMask or hardware wallet.")]
+    private let cards = [
```

Comment:
I think it'd be good to declare the tuple type here for better code readability:

```suggestion
    private let cards: [(image: UIImage, title: String, body: String, link: (title: String, url: URL)?)] = [
```

Reply:
ok